### PR TITLE
support aarch64

### PR DESCRIPTION
**Pull request by rink1969**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,11 +4,19 @@ CARGO=RUSTFLAGS='-F warnings -A deprecated' cargo
...
-	scripts/release.sh release
+	scripts/release.sh x86 release
+
+aarch64_debug:
+	$(CARGO) build --all --target aarch64-unknown-linux-gnu
+	scripts/release.sh aarch64 debug
+
+aarch64_release:
+	$(CARGO) build --all  --release --target aarch64-unknown-linux-gnu
+	scripts/release.sh aarch64 release
```
Comment (leeyr338) [inline Makefile:19]: `aarch64` may has different instruction set.
Is it better to use instruction arch (something like A57, A7) ?

File: aarch64.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
+# 在华为鲲鹏上运行CITA
+
+鲲鹏是华为海思发布的基于ARM架构授权，由华为自主设计完成的鲲鹏芯片。
```
Comment (leeyr338) [inline aarch64.md]: Needs `space` between Chinese and English.

File: Makefile (hunk 1 above)
Comment (rink1969) [reply Makefile:19]: 指令集是arm v8，只有v8才支持64位。
用这个是因为交叉编译器以及rust对应的target都是用这个开头的。

File: aarch64.md (hunk 2 above)
Comment (rink1969) [reply aarch64.md]: ok

File: Makefile (hunk 1 above)
Comment (leeyr338) [reply Makefile:19]: OK

**Review by leeyr338 — APPROVED** [review]

**Pull request merged**